Sources/Sentry/SentryUIPerformanceTracker.m

```diff
@@ -0,0 +1,239 @@
+#import "SentryUIPerformanceTracker.h"
+#import "SentryHub.h"
+#import "SentryLog.h"
+#import "SentryPerformanceTracker.h"
+#import "SentrySDK+Private.h"
+#import "SentryScope.h"
+#import "SentrySpanId.h"
+#import "SentrySwizzle.h"
+#import "SentryUIViewControllerSanitizer.h"
+#import <objc/runtime.h>
+
+#if SENTRY_HAS_UIKIT
+#    import <UIKit/UIKit.h>
+#endif
+
+static NSString *const SENTRY_UI_PERFORMANCE_TRACKER_SPAN_ID
+    = @"SENTRY_UI_PERFORMANCE_TRACKER_SPAN_ID";
+
+static NSString *const SENTRY_UI_PERFORMANCE_TRACKER_LAYOUTSUBVIEW_SPAN_ID
+    = @"SENTRY_UI_PERFORMANCE_TRACKER_LAYOUTSUBVIEW_SPAN_ID";
+
+static NSString *const SENTRY_VIEWCONTROLLER_RENDERING_OPERATION = @"ui.rendering";
```

Comment:
M: we'd need to align those things, Android and RN are `navigation`

Reply:
I had a meeting with @bruno-garcia and he think using "navigation" is wrong.
"ui.rendering" is temporary until someone defines the correct one.

Reply:
Let's deal with this here https://github.com/getsentry/sentry-cocoa/issues/1129 and don't block this PR with that.